src/main/java/org/folio/service/BaseService.java

```diff
@@ -114,20 +112,21 @@ public CompletableFuture<String> handlePostRequest(JsonObject recordData, String
     CompletableFuture<String> future = new CompletableFuture<>();
     try {
       if (logger.isDebugEnabled()) {
-        logger.debug("Sending 'POST {}' with body: {}", endpoint, recordData.encodePrettily());
+        logger.debug("Trying to create object by endpoint '{}' and body '{}'", endpoint, recordData.encodePrettily());
       }
       httpClient.request(HttpMethod.POST, recordData.toBuffer(), endpoint, okapiHeaders)
         .thenApply(this::verifyAndExtractRecordId)
         .thenAccept(id -> {
           future.complete(id);
-          logger.debug("'POST {}' request successfully processed. Record with '{}' id has been created", endpoint, id);
+          logger.debug("Object was successfully created. Record with '{}' id has been created", id);
         })
```

Comment:
which one is more informative 
'POST {}' request su...
Object was.....

Reply:
@azizbekxm  the same structure as you did in mod-gobi, so lets use second one

Reply:
ok, Thanks